anyway/parsers/injured_around_schools_2022.py

```diff
@@ -0,0 +1,560 @@
+    baseY = lat_min
+    distanceX = lon_max
+    distanceY = lat_max
+    pol_str = "POLYGON(({0} {1},{0} {3},{2} {3},{2} {1},{0} {1}))".format(
+        baseX, baseY, distanceX, distanceY
+    )
+
+    query_obj = (
+        db.session.query(InvolvedMarkerView)
+        .filter(InvolvedMarkerView.geom.intersects(pol_str))
+        .filter(
+            or_(
+                (InvolvedMarkerView.provider_code == BE_CONST.CBS_ACCIDENT_TYPE_1_CODE),
+                (InvolvedMarkerView.provider_code == BE_CONST.CBS_ACCIDENT_TYPE_3_CODE),
+            )
+        )
+        .filter(InvolvedMarkerView.accident_timestamp >= start_date)
+        .filter(InvolvedMarkerView.accident_timestamp < end_date)
+        .filter(InvolvedMarkerView.location_accuracy.in_(LOCATION_ACCURACY_PRECISE_LIST))
+        .filter(InvolvedMarkerView.age_group.in_(AGE_GROUPS))
+        .filter(InvolvedMarkerView.injury_severity.in_([1, 2, 3]))
+        .filter(
+            or_(
+                (InvolvedMarkerView.injured_type.in_(INJURED_TYPES)),
+                (InvolvedMarkerView.involve_vehicle_type.in_(VEHICLE_TYPES))
+            )
+        )
+        .filter(
+            InvolvedMarkerView.accident_hour_raw.between(SEVEN_AM_RAW, SEVEN_PM_RAW)
+        )
+        # .with_entities(InvolvedMarkerView.geom,
+        #                InvolvedMarkerView.injured_type,
+        #                InvolvedMarkerView.injured_type_hebrew,
+        #                InvolvedMarkerView.involve_vehicle_type,
+        #                InvolvedMarkerView.involve_vehicle_type_hebrew,
+        #                InvolvedMarkerView.injury_severity,
+        #                InvolvedMarkerView.injury_severity_hebrew,
+        #                InvolvedMarkerView.speed_limit,
+        #                InvolvedMarkerView.speed_limit_hebrew,
+        #                InvolvedMarkerView.provider_code,
+        #                InvolvedMarkerView.accident_timestamp,
+        #                InvolvedMarkerView.location_accuracy,
+        #                InvolvedMarkerView.age_group,
+        #                InvolvedMarkerView.age_group_hebrew,
+        #                InvolvedMarkerView.accident_year,
+        #                InvolvedMarkerView.injury_severity,
+        #                InvolvedMarkerView.cross_location,
+        #                InvolvedMarkerView.cross_location_hebrew,
+        #                )
+    )
+
+    df = pd.read_sql_query(query_obj.statement, query_obj.session.bind)
+
+    if LOCATION_ACCURACY_PRECISE:
+        location_accurate = 1
+        location_approx = ""
+    else:
+        location_accurate = 1
+        location_approx = 1
+
+    ui_url_map_only = ANYWAY_UI_FORMAT_MAP_ONLY.format(
+        latitude=latitude,
+        longitude=longitude,
+        start_date=start_date.strftime(DATE_URL_FORMAT),
+        end_date=end_date.strftime(DATE_URL_FORMAT),
+        acc_type=SUBTYPE_ACCIDENT_WITH_PEDESTRIAN,
+        location_accurate=location_accurate,
```

Comment:
Here we actually changed the logic so we want to make sure the links will be aligned with it as well.
I suggest the following:
- remove location accuracy from the links
- Filter in `models.py` where light_transportation == True (line 613)